unittests/BuildSystem/BuildSystemTaskTests.cpp

```diff
@@ -408,8 +408,8 @@ TEST(BuildSystemTaskTests, staleFileRemoval) {
     "commandPreparing(C.1)",
     "commandStarted(C.1)",
     // FIXME: Maybe it's worth creating a virtual FileSystem implementation and checking if `remove` has been called
-    "cannot remove stale file 'a.out': No such file or directory",
-    "commandFinished(C.1: 1)",
+    "commandWarning(C.1) cannot remove stale file 'a.out': No such file or directory\n",
+    "commandFinished(C.1: 0)",
```

Comment:
Just as a note, all these commands are now successful, since we demoted the messages to warnings.